filament/backend/src/metal/MetalDriver.mm

```diff
@@ -1354,15 +1354,15 @@
         return mContext->samplerStateCache.getOrCreateState(s);
     };
 
-    id<MTLTexture> texturesToBindVertex[MAX_VERTEX_SAMPLER_COUNT] = {};
-    id<MTLSamplerState> samplersToBindVertex[MAX_VERTEX_SAMPLER_COUNT] = {};
+    id<MTLTexture> texturesToBindVertex[FEATURE_CAPS[1].MAX_VERTEX_SAMPLER_COUNT] = {};
```

Comment:
Use `FEATURE_CAPS[+FeatureLevel::FEATURE_LEVEL_1]` to be consistent with SamplerBindingMap.cpp?

Reply:
done